lib/reek/core/smell_repository.rb

```diff
@@ -13,12 +13,17 @@ def self.smell_types
         Reek::Smells::SmellDetector.descendants
       end
 
+      def self.default
```

Comment:
This may be a little confusing since in the Singleton mixin this would always return the same instance.